pkg/networkmanager/index.html

```diff
@@ -43,7 +43,7 @@ <h1 id="testing-connection-curtain-restoring" translatable="yes">Restoring conne
     <div id="networking-nm-crashed" class="alert alert-danger" hidden>
       <button class="btn btn-default pull-right" type="submit" translatable="yes">Start Service</button>
         <span class="pficon pficon-error-circle-o"></span>
-        <strong translatable="yes">NetworkManager is not running.</strong> <a href="#" class="alert-link">Troubleshoot…</a>
+        <strong translatable="yes">NetworkManager is not running.</strong> <a tabindex="0" href="#" class="alert-link">Troubleshoot…</a>
```

Comment:
Oops. Somehow an href got in the mix. We should remove the tabindex here.

```suggestion
        <strong translatable="yes">NetworkManager is not running.</strong> <a href="#" class="alert-link">Troubleshoot…</a>
```